app/i18n/global-messages.js

```diff
@@ -249,3 +270,12 @@ export const environmentSpecificMessages = {
     },
   }),
 };
+
+export function listOfTranslators(contributorsList, contributorsAck): string {
```

Comment:
@leivaburto 

- if you make a separate component them this whole function `listOfTranslators` can be moved to that component, don't feel this function should be here.

- `if (contributorsList !== globalMessages.translationContributors.defaultMessage)`
comparing to `defaultMessage` will get you in trouble if we decide to use language specific names (that still need to decided), but we never use `defaultMessage` for logical things.
I feel, the whole if condition is not needed here, `if (contributorsList !== globalMessages.translationContributors.defaultMessage)`

Reply:
Regarding your second comment, the idea is this: there would be an empty string, and contributors who are willing to do so will put their names in. Empty string messages fallback to defaultMessage value, so contributorsList would take that value. What do you mean by using language specific names? AFAIK strings would fallback to default values defined in global messages, regardless of language, which in this case is `!!!List of contributors`. Or is there another strategy to be considered?

Reply:
Regarding your first comment, it makes sense. Actually, I was working on PR #494 and I had to make something similar, so I'm planning to use that same component in here (it's called InformativeMessage). The component is generic, so I'm trying to figure out where would be better to put the method you mention. Anyway, I want to make sure the component I added makes sense (I think it does). Could you take a look at it? and if everything is OK I could use it here. Btw, thanks for your feedback.

Reply:
@leivaburto 
> What do you mean by using language specific names?
Got clear after your explanation.